src/jimgw/core/prior.py

```
@@ -21,58 +21,92 @@
+        Args:
+            z: Dict of parameter values.
+
+        Returns:
+            Log-probability scalar.
+        """
         raise NotImplementedError
 
     @abstractmethod
     def sample(
         self, rng_key: Key, n_samples: int
     ) -> dict[str, Float[Array, " n_samples"]]:
```

Ruff (0.15.15)

[error] Syntax error in forward annotation: Expected an expression

(F722)